crmsh/bootstrap.py

```diff
@@ -2731,12 +2733,15 @@ def bootstrap_init_geo(context):
 
 
 def geo_fetch_config(node):
+    print(node)
     user, node = utils.parse_user_at_host(node)
+    print(user)
     if user is not None:
         try:
             local_user = utils.user_of(utils.this_node())
         except utils.UserOfHost.UserNotFoundError:
-            local_user = userdir.getuser()
+            local_user = user
+        print(local_user)
```

Comment:
These 3 `print` are here on purpose, or forget to remove?

Reply:
They are forgotten.